Makefile

```diff
@@ -124,6 +124,12 @@ endif
 TEXT_ENGINE := none
 $(eval $(call validate-option,TEXT_ENGINE,none s2dex_text_engine))
 
+ifeq ($(TEXT_ENGINE), s2dex_text_engine)
+  DEFINES += S2DEX_GBI_2=1 S2DEX_TEXT_ENGINE=1
+  SRC_DIRS += src/s2d_engine
+endif
+# add more text engines here
+
```

Comment:
why was this moved

Reply:
`SRC_DIRS` won't populate correctly unless this happens before `C_FILES` is set